lib/src/Uncertainty/Distribution/TruncatedNormal.cxx

```diff
@@ -472,11 +559,11 @@ Bool TruncatedNormal::isElliptical() const
 void TruncatedNormal::setMu(const Scalar mu)
 {
   if (mu != mu_)
-  {
-    mu_ = mu;
-    isAlreadyComputedMean_ = false;
-    isAlreadyComputedCovariance_ = false;
-  }
+    {
+      mu_ = mu;
+      isAlreadyComputedMean_ = false;
+      isAlreadyComputedCovariance_ = false;
+    }
```

Comment:
The original indentation seems clearer to me: why is the first { indented with respect to "if" ? The same goes for other places.

Reply:
@josephmure This is because the default indentation used by emacs when I write the code is not the same as the one we impose in the project. I have a macro to call explicitly within emacs, and sometimes i forget to do this. As it is done systematically before each release it is not a so big deal...

Reply:
I didn't know we had a script to automatically correct the indentation before a release, that's great!

Reply:
@josephmure The script is [here](https://github.com/openturns/openturns/blob/master/utils/indent.sh)